docs/advanced-features/athena.ipynb

```diff
@@ -0,0 +1,662 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "fe895e07-0f59-4803-9600-fe7fa5b30d1f",
+   "metadata": {},
+   "source": [
+    "# Querying package metadata with Athena\n",
+    "Quilt stores package data and metadata in S3. Metadata lives in a per-package manifest file in each bucket's `.quilt/` directory.\n",
+    "\n",
+    "You can therefore query package metadata wth SQL engines like AWS Athena.\n",
+    "Users can write SQL queries to select packages (or files from within packages)\n",
+    "using predicates based on package or object-level metadata."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "73600303-002f-4460-b0cc-7176fd6f55e1",
+   "metadata": {},
+   "source": [
+    "## Note: Executing Documentation Code  \n",
+    "If you import your AWS credentials for use by `boto3`, ",
+    "you can edit and execute code directly from the notebook version of this document.\n",
+    "You can alternatively copy and paste it into your Python editor.\n",
+    "<!--pytest.mark.skip-->"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "5e7ea237-20bc-470a-b009-047a93d833b9",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%capture\n",
+    "#%env AWS_PROFILE=default\n",
+    "%pip install boto3"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5125f6c0-b75d-4230-a81e-63f91073d091",
+   "metadata": {},
+   "source": [
+    "This allows you to configure AWS services by calling Python objects:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "c90a031d-3d71-48a2-95e4-3cbb7923375c",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Session(region_name='us-east-1')\n"
+     ]
+    }
+   ],
+   "source": [
+    "import boto3,json,re,time\n",
+    "SESSION = boto3.session.Session()\n",
+    "print(SESSION)\n",
+    "REGION=SESSION.region_name\n",
```

Comment:
Example should follow [PEP 8](https://peps.python.org/pep-0008/), consider using [black](https://black.readthedocs.io/en/stable/getting_started.html#installation).

Reply:
Updated to match recommendations from [Black](https://black.vercel.app/?version=stable&state=_Td6WFoAAATm1rRGAgAhARYAAAB0L-Wj4ADNAItdAD2IimZxl1N_WlwxRhR7MSgEGOu7V_6TXAxr41EM8YRC5L795qahpcAux9lOcS3gZoJYWh6YNFRU85_wdrDByU_N0RVOpnehqcJksSnu3d2P5UnjSWokUMffMJtJ1-TLeFZit7qD7z8AaxbJZFxfhi28DzpL31ETmeO42nUEmXbQFzbup2ePXzzXpwAAAMviOKt51rwhAAGnAc4BAAA5L3xtscRn-wIAAAAABFla)

Reply:
I mean *all* examples, i.e. run `black docs/advanced-features/athena.ipynb`.

Reply:
DONE: poetry run black ../docs/advanced-features/athena.ipynb